docs/decisions/006-experimental-code.md

```diff
@@ -0,0 +1,41 @@
+# Experimental Code
+
+Date: 2024-06-03
+
+Status: proposed
+
+## Context
+
+The team occasionally authors code, or accepts contributions, that are considered experimental or unstable.
+The goal for this code is to ship it as unstable for sponsor product to leverage.
+During this time, the team can get feedback around what is working and what does not work so that changes can be made before an official release.
```

Comment:
```suggestion
During this time, the team can get feedback on what is working and what does not work so that changes can be made before an official release.
```

Reply:
Fixed.